qiskit_pkg/setup.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2018.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+# This file is the setup.py file for the qiskit package. Because python
+# packaging doesn't offer a mechanism to have qiskit supersede qiskit-terra
+# and cleanly upgrade from one to the other, there needs to be a separate
+# package always needs
+
+import json
+import os
+
+from setuptools import setup
+
+README_PATH = os.path.join(os.path.abspath(os.path.dirname(__file__)), "README.md")
+with open(README_PATH) as readme_file:
+    README = readme_file.read()
+
+EXTRAS_PATH = os.path.join(os.path.abspath(os.path.dirname(__file__)), "extras.json")
+with open(EXTRAS_PATH) as fd:
+    extras_dict = json.load(fd)
+
+extras_dict["all"] = [pkg for requirements in extras_dict.values() for pkg in requirements]
+
+requirements = ["qiskit-terra==0.45.0"]
+
+setup(
+    name="qiskit",
+    version="0.45.0",
+    description="Software for developing quantum computing programs",
+    long_description=README,
```

Comment:
So this will be the string `"../README.md"`, right? Does `setup` take either a file path or the contents directly and decide what to do from there? Maybe I'm missing something 😄

Reply:
Those files are actually symlinks in the repo, that's just how GitHub displays them.  No idea how Windows will handle them, tbh.

Reply:
It apparently works on newer versions of windows: https://stackoverflow.com/questions/5917249/git-symbolic-links-in-windows/59761201#59761201